src/chat.js

```diff
@@ -0,0 +1,39 @@
+import io from 'socket.io-client';
+import 'jquery';
+
+export default class Chat {
+    constructor(nick) {
+        this.nick = nick;
+        this.socket = io({query: `nick=${nick}`});
+
+        if (__DEV__)
+            console.log('nick: '+nick);
+
+        this.socket.on('pong', (socket) => {
+            if (__DEV__)
+                console.log('pong: '+ socket);
+        });
+
+        this.socket.on('serverMessage', (data) => {
+            this.addMessage(data.nick, data.message);
+        });
+
+        this.chatInput = $('#chatInput').get(0);
+        this.chatBox = $('#chatBox').get(0);
+
+        this.chatInput.addEventListener('keypress', (key) => {
```

Comment:
I would use the Jquery version of this event listener.
`this.chatinput.keypress((key) => {});` or
`this.chatinput.on('keypress', (key) => {});`